Doc/library/json.rst

```diff
@@ -60,6 +60,17 @@ Pretty printing::
         "6": 7
     }
 
+Specializing JSON object encoding::
```

Comment:
It's out of scope for this PR, but while you're here what do you think of changing the example immediately above to encode `{'6': 7, '4': 5}` instead, so that it better showcases that `sort_keys` sorts the keys?

Reply:
Done! Good idea